docs/tutorial_advanced.rst

```diff
@@ -292,13 +279,59 @@ The following test is a compile-only version of the :class:`MakefileTest` presen
 
 
 .. literalinclude:: ../tutorials/advanced/makefiles/maketest.py
-   :lines: 25-33
+   :lines: 27-35
    :emphasize-lines: 2
 
-What is worth noting here is that the standard output and standard error of the test, which are accessible through the :attr:`stdout <reframe.core.pipeline.RegressionTest.stdout>` and :attr:`stderr <reframe.core.pipeline.RegressionTest.stderr>` attributes, correspond now to the standard output and error of the compilation command.
+What is worth noting here is that the standard output and standard error of the test, which are accessible through the :attr:`~reframe.core.pipeline.RegressionTest.stdout` and :attr:`~reframe.core.pipeline.RegressionTest.stderr` attributes, correspond now to the standard output and error of the compilation command.
 Therefore sanity checking can be done in exactly the same way as with a normal test.
 
 
+Grouping parameter packs
+------------------------
+
+.. versionadded:: 3.4.2
+
+
+In the dot product example shown above, we had two independent tests that defined the same :attr:`elem_type` parameter.
+And the two tests cannot have a parent-child relationship, since one of them is a run-only tests and the other is a compile-only one.
```

Comment:
```suggestion
And the two tests cannot have a parent-child relationship, since one of them is a run-only test and the other is a compile-only one.
```